docs/GettingStarted/configuration/ExporterWebhook.md

```diff
@@ -0,0 +1,203 @@
+# Webhook Exporter
+
+The Webhook exporter is capable of receiving data from any service that sends an appropriate HTTP POST request.
+
+The HTTP POST request consists of two parts, [Header and the Payload](#webhook-headers-and-payloads). The Header data specifies which webhook plugin should be used to validate and transform the payload into a Pelorus metric, that is consumed by Prometheus.
```

Comment:
isn't "Model" instead of "plugin" in this line?

Reply:
it should be "plugin". The "Model" is an internal implementation detail here that applies to the plugin.